modulefiles/build.wcoss_dell_p3

```diff
@@ -1,25 +1,31 @@
 #%Module#####################################################
-## global_chgres component - wcoss_dell_p3
+## Build and run module for WCOSS-Dell P3
 #############################################################
+
+module load lsf/10.1
+module load HPSS/5.0.2.5
+module load cmake/3.16.2
 module load ips/18.0.1.163
 module load impi/18.0.1
-
 module load w3nco/2.0.6
 module load w3emc/2.3.0
 module load sp/2.0.2
 module load nemsio/2.2.3
-module load nemsiogfs/2.0.1
+module load bacio/2.0.2
 module load ip/3.0.1
 module load sfcio/1.0.0
+module load sigio/2.1.0
 module load gfsio/1.1.0
 module load landsfcutil/2.1.0
-module load bacio/2.0.2
-module load  sigio/2.1.0 
+module load nemsiogfs/2.0.1
+module load g2/3.1.0
+module load jasper/1.900.29
+export Jasper_ROOT="/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29"
```

Comment:
what does `module load jasper/1.900.29` do if you have to manually export `Jasper_ROOT`?
If the module loads an environment variable e.g. `JASPER`, it is best to use that to set it to `Jasper_ROOT`, rather than specifying the full path.

Reply:
This is going to be harder than simply setting the variable because of it being in a module file. We should ask the sysadmin to define this variable in the jasper modulefile itself.
I take this comment back.

Reply:
Here is the output from the Dell module.  Many of our modules don't provide what CMake wants.  And I don't always know the best way to handle it.  Should I not load the module at all and just hardwire what I need?  Should I try to create Jasper_ROOT from one of the exiting variables?

```prepend_path("PATH","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/bin")
prepend_path("LD_LIBRARY_PATH","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/lib")
prepend_path("MANPATH","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29//man")
setenv("JASPER_INC","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/include")
setenv("JASPER_LIBDIR","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/lib")
setenv("JASPER_LIB","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/lib/libjasper.a")
setenv("JASPER_SRC","/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29/src/jasper-1.900.29")
setenv("JASPER_VER","1.900.29")

Reply:
@GeorgeGayno-NOAA Leave as is for now and open a ticket with WCOSS helpdesk and ask them to add a line in their modulefile for Jasper as follows to allow CMake to find the correct Jasper installation:
```
setenv("Jasper_ROOT", "/usrx/local/prod/packages/gnu/4.8.5/jasper/1.900.29")
```
When they agree and make the change, we can remove the `export Jasper_ROOT= ...` line from our modulefile.